packages/fxa-auth-server/lib/error.js

```diff
@@ -1073,24 +1073,22 @@ AppError.unknownSubscriptionPlan = (planId) => {
   });
 };
 
-AppError.rejectedSubscriptionPaymentToken = (token, message) => {
+AppError.rejectedSubscriptionPaymentToken = (message, paymentError) => {
   return new AppError({
     code: 400,
     error: 'Bad Request',
     errno: ERRNO.REJECTED_SUBSCRIPTION_PAYMENT_TOKEN,
     message
-  }, {
-    token
-  });
+  }, paymentError);
 };
 
-AppError.rejectedCustomerUpdate = (message) => {
+AppError.rejectedCustomerUpdate = (message, paymentError) => {
   return new AppError({
     code: 400,
     error: 'Bad Request',
     errno: ERRNO.REJECTED_CUSTOMER_UPDATE,
```

Comment:
Tangent: I noticed this and `SUBSCRIPTION_ALREADY_CANCELLED` are missing from the content server's `app/scripts/lib/auth-errors.js`, have opened #1602.